sdk/communication/AzureCommunication/Package.swift

```diff
@@ -0,0 +1,69 @@
+// swift-tools-version:5.3
+//  The swift-tools-version declares the minimum version of Swift required to build this package.
+//
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import PackageDescription
+
+let package = Package(
+    name: "AzureCommunication",
+    platforms: [
+        .macOS(.v10_13), .iOS(.v12)
+    ],
+    products: [
+        .library(name: "AzureCommunication", targets: ["AzureCommunication"])
+    ],
+    dependencies: [],
+    targets: [
+        // Build targets
+        .target(
+            name: "AzureCommunication",
+            dependencies: ["AzureCore"],
```

Comment:
I don't understand swift packages that much but don't you need to list a version and/or repository? Does this essentially take a source level dependency the way it is currently configured?

Reply:
No the git reference is the version. Swift is 100% tied to Git as a version control mechanism.

Reply:
The target dependencies reference the package dependencies by name. The package dependencies will reference the repos (in this case, the SwiftPM-* repos) and either a version, commit, or branch.

Reply:
Where does that repo name and tag info get stored for these dependencies?

Reply:
I see that information listed under the dependencies section above now.

Reply:
Here's the example from Chat

let package = Package(
    name: "AzureCommunicationChat",
    platforms: [
        .iOS(.v12)
    ],
    products: [
        .library(name: "AzureCommunicationChat", targets: ["AzureCommunicationChat"])
    ],
    dependencies: [
        **.package(name: "AzureCore", url: "https://github.com/Azure/SwiftPM-AzureCore.git", .branch("master")),**
    ],
    targets: [
        // Build targets
        .target(
            name: "AzureCommunicationChat",
            **dependencies: ["AzureCore"],**
            path: "Source",
            exclude: [
                "README.md",
                "Tests",
                "Source/Supporting Files",
                "LICENSE"
            ]
        ),
...